mmv1/third_party/terraform/services/compute/resource_compute_router_bgp_peer_test.go.tmpl

```diff
@@ -248,6 +248,30 @@ func TestAccComputeRouterPeer_Ipv4BasicCreateUpdate(t *testing.T) {
 	})
 }
 
+func TestAccComputeRouterPeer_UpdateRouterCustomLearnedRoutePriority(t *testing.T) {
```

Comment:
Can you add a step to this test that sets the priority to 0 but sets the boolean to false? We want to make sure that functionality works too.

Reply:
i change the debug statement to error and it will be decided by customer which they have to assign it .

Reply:
Ok great. What happens when the boolean is false/unset and the field is unset? I'm worried this will effectively require the user to set the field.

We should make sure that case works